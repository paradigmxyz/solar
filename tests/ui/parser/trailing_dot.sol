uint256 constant a = 2 + 2.;
//~^ ERROR: empty rational
