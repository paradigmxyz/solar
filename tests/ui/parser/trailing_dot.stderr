error: empty rational
  --> ROOT/tests/ui/parser/trailing_dot.sol:LL:CC
   |
LL | uint256 constant a = 2 + 2.;
   |                          ^^
   |

error: aborting due to 1 previous error

